### PR TITLE
TRAFODION 1547 and TRAFODION 1476

### PR DESCRIPTION
**Pull request by amandamoran**

### CODE REVIEW

File: install/installer/traf_add_user [hunk 1]
```diff
@@ -208,15 +208,26 @@ fi
...
+NODES_SHORT=""
 for node in $NODE_LIST
-do
-    sudo su $TRAF_USER --command "ssh -q -oStrictHostKeyChecking=no $node hostname"
-    if [ $? -ne 0 ]; then
-        echo "***ERROR: Unable to ssh to node $node"
-        exit -1
-    fi
+do 
+   newNode=$(ssh -q -n $node hostname -s)
```
Comment (arvind-narain) [inline install/installer/traf_add_user]: should the passwordless ssh for $NODE_LIST be setup first?

Comment (amandamoran) [inline install/installer/traf_add_user]: Yes, the user needs to set up passwordless ssh and there is a check for that earlier in traf_config_check

**Comment by amandamoran** [comment]
jenkins, retest

**Comment by arvind-narain** [comment]
Thanks Amanda

**Comment by amandamoran** [comment]
jenkins, retest

**Comment by DaveBirdsall** [comment]
Looks good to me. Is this ready to commit?

**Comment by amandamoran** [comment]
Yes, please :)

**Comment by DaveBirdsall** [comment]
+1



*From:* Amanda [mailto:notifications@github.com]
*Sent:* Monday, December 14, 2015 12:27 PM
*To:* apache/incubator-trafodion <incubator-trafodion@noreply.github.com>
*Cc:* DaveBirdsall <dave.birdsall@esgyn.com>
*Subject:* Re: [incubator-trafodion] TRAFODION 1547 and TRAFODION 1476
(#217)



Yes, please :)

—
Reply to this email directly or view it on GitHub
<https://github.com/apache/incubator-trafodion/pull/217#issuecomment-164549577>
.

**Comment by arvind-narain** [comment]
Thanks Amanda - I'll merge this change.

**Pull request merged**